packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/index.md

```diff
@@ -9,24 +9,26 @@ meta:
 
 This document explains how to configure <StackSelector snippet="idp" noSelector inline /> as an external Identity Provider for your application by creating an application at <StackSelector snippet="idp" noSelector inline />, creating an Identity Provider in Okta, testing the configuration, and creating a sign-in button.
```

Comment:
```suggestion
This document explains how to configure <StackSelector snippet="idp" noSelector inline /> as an external Identity Provider for your app by creating an app at <StackSelector snippet="idp" noSelector inline />, creating an Identity Provider in Okta, testing the configuration, and creating a sign-in button.
```

Reply:
I interpret the InfoDev style guide to say we should shorten "application" to "app", in most cases.

"In general, use [app] instead of application."